src/tickit/core/simulation.py

```diff
@@ -87,20 +90,20 @@ def build_simulation(
     """
     scheduler = None
     components = None
-    components_to_run = components_to_run or set()
-
     configs = read_configs(config_path)
-    inverse_wiring = InverseWiring.from_component_configs(configs)
 
     if include_schedulers:
+        inverse_wiring = InverseWiring.from_component_configs(configs)
         scheduler = MasterScheduler(inverse_wiring, *get_interface(backend))
+
     if include_components:
-        components = {config.name: config() for config in configs}
-        run_all = not components_to_run
+        if components_to_run == ("all",) or components_to_run is None:
```

Comment:
When would `components_to_run` ever be a tuple?

Reply:
I don't know why this does this, but click spits it out like this.